integration-tests/src/driver/mod.rs

```diff
@@ -0,0 +1,327 @@
+mod example;
+
+use crate::polkadot_test_net::*;
+use frame_support::assert_ok;
+use frame_support::traits::fungible::Mutate;
+use frame_support::BoundedVec;
+use hydradx_runtime::*;
+use hydradx_traits::stableswap::AssetAmount;
+use hydradx_traits::AggregatedPriceOracle;
+use pallet_asset_registry::AssetType;
+use pallet_stableswap::MAX_ASSETS_IN_POOL;
+use primitives::constants::chain::{OMNIPOOL_SOURCE, STABLESWAP_SOURCE};
+use primitives::{AccountId, AssetId};
+use sp_runtime::{FixedU128, Permill};
+use xcm_emulator::TestExt;
+
+pub(crate) struct HydrationTestDriver {
+	omnipool_assets: Vec<AssetId>,
+	stablepools: Vec<(AssetId, Vec<(AssetId, u8)>)>,
+}
+
+impl HydrationTestDriver {
+	pub(crate) fn add_omnipool_assets(self, assets: Vec<AssetId>) -> Self {
```

Comment:
any reason that there are two implementation block for HydrationTestDriver?

Reply:
no reason. i can merge it